answer_files/10/Autounattend.xml

```diff
@@ -2,6 +2,28 @@
 <unattend xmlns="urn:schemas-microsoft-com:unattend">
     <servicing/>
     <settings pass="windowsPE">
+        <component name="Microsoft-Windows-PnpCustomizationsWinPE"
+            publicKeyToken="31bf3856ad364e35" language="neutral"
+            versionScope="nonSxS" processorArchitecture="amd64"
+            xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State">
+
+            <!--
+                 This makes the VirtIO drivers available to Windows, assuming that
+                 the VirtIO driver disk
+                 (https://alt.fedoraproject.org/pub/alt/virtio-win/latest/images/bin/)
+                 is available as drive E:
+            -->
+            <DriverPaths>
+                <PathAndCredentials wcm:action="add" wcm:keyValue="2">
+                    <Path>E:\viostor\w7\amd64</Path>
```

Comment:
Will this make problems for the other packer builders like VMware or Virtualbox if the E: drive isn't there?

Reply:
I gave that a try, by adding an invalid path - like `K:\Kaboom\w7\amd64`. Setup did not fail, so I guess invalid/non-existent paths are just silently ignored.